book-worm-2010/book-worm-fe
# Tm/feature/modal

### What does this PR do? 
>Clearly and concisely describe the feature. 
This PR adds styling to the homepage adding in images as button that are/can become clickable.

### Is this a feature or Fix/Refactor?  
Feature

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
src/Home/Home.js;
src/Home/_Home.scss;
src/NewBookForm/_NewBookForm.scss;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. run `npm start`
2. Click on the images or hover over them to see the effects

### Steps To Take
>Any questions and possible steps moving forward.
Will need to decide what else should be clickable and if we need to add more buttons.
